documentation/cmake-bootstrap.md

```diff
@@ -45,7 +45,17 @@ for validation, set the CMake cache variable `FINAL_STAGE` to `2`.
 
 Jakt is known to compile with clang >=13 on Linux, macOS and Windows. g++ also works, provided the version is >=10.2.
 
-MSVC is not supported, however clang-cl.exe and clang.exe do work and clang-cl is used in CI.
+MSVC is not supported, however clang-cl.exe and clang.exe do work and clang-cl is used in CI. You must specify `-GNinja`, as CMake may otherwise default to MSVC generators which will fail. Additionally, clang-cl seems dependent on environment variables to find MSVC, so ensure you're running it from the Developer Command Prompt. Clang++ doesn't seem to have the same requirement.
```

Comment:
Developer command prompt or developer powershell work equally well. I have only tried with VS 2022 Community Preview, I'm not sure what version of VS 2022 is the newest that will work.,  (on 17.7.0 preview rn).